### DISCUSSION TITLE
I can't fork a repo with the same prefix existing in my github

### DISCUSSION THREAD

**Discussion by tuan-karma** [community/community#60091] [Repositories]
### Select Topic Area

Bug

### Body

The context:
- I've forked a repo with name: https://github.com/tuan-karma/the-forked-repo
- Now, I want to fork another repo with name: https://github.com/tuan-karma/the-forked-repo-with-extension

The github webpage not alow me to do that. It indicated that: "fork already exist" when I am trying to fork the later repo. 

My temporary solution now: I have to delete the former forked repo (that actually a different repo from another author) :(

To the github team: Please fix this bug soon. 
Thank you very much

**Comment by GhostOf0days** [comment]
Another solution rather than deleting the other forked repo would be to rename it in the forked repo's settings, fork the new repo, then rename the other repo back.

**Comment by thescientist13** [comment]
Yes, just ran into the issue.

I have a "baseline" repo from which I would like to spin off many variants from it, similar to your case, e.g.
- baseline
- baseline-integration-A
- baseline-integration-B
- etc
